pkg/chart/repo/repo.go

```diff
@@ -224,7 +224,14 @@ func (r *Repo) FetchRemote(cv *repo.ChartVersion) (*chart.Chart, error) {
 	url := chartURL.String()
 	data, err := r.fetcher(url)
 	if err != nil {
-		return nil, err
+		chartspec, convErr := chartVerToChartSpec(cv)
```

Comment:
Renaming `chartVerToChartSpec` to `newChart` makes more sense in here, and renaming the variables as well:

```go
chart, newErr := newChart(cv)
```